pyfolio/tests/test_txn.py

```diff
@@ -41,25 +36,15 @@ def test_get_turnover(self):
         result = get_turnover(positions, transactions)
         assert_series_equal(result, expected)
 
-        # Monthly freq
-        index = date_range('01-01-2015', freq='M', periods=1)
-        expected = Series([0.0], index=index)
-        result = get_turnover(positions, transactions, period='M')
-        assert_series_equal(result, expected)
-
-        transactions = DataFrame(data=[[1, 1, 10, 'A']]*len(dates),
+        transactions = DataFrame(data=[[1, 1, 10, 'A']]*len(dates) +
+                                 [[2, -1, 10, 'B']]*len(dates),
                                  columns=['sid', 'amount', 'price', 'symbol'],
-                                 index=dates)
+                                 index=dates.append(dates)).sort_index()
 
-        expected = Series([0.5]*len(dates), index=dates)
+        expected = Series([4.0] + [2.0] * (len(dates) - 1), index=dates)
         result = get_turnover(positions, transactions)
         assert_series_equal(result, expected)
```

Comment:
A couple comments on this test:
- it's probably worth adding a test for the case where you pass `denominator='portfolio_value'`
- the `positions` DataFrame confuses me a bit here. If there are transactions for 2 different sids, shouldn't the `positions` DataFrame presumably have columns for 2 sids as well? Also worth noting that the `positions` column for sid `0` is set to always be 10.0, meaning that doesn't really capture the averaging behavior where you do `AGB.rolling(2).mean()`. Incremental or alternating positions values would remedy this.

Reply:
Good idea, I'll add this, and change it to just 1 sid.